packages/react-components/src/components/MessageStatusIndicator.tsx

```diff
@@ -116,6 +118,7 @@ export const MessageStatusIndicator = (props: MessageStatusIndicatorProps): JSX.
           data-ui-id="chat-composite-message-tooltip"
           calloutProps={{ ...calloutProps }}
         >
+          <LiveMessage message={strings.sendingAriaLabel ?? ''} aria-live="polite" />
           <Icon
             role="status"
             data-ui-id="chat-composite-message-status-icon"
```

Comment:
A few concerns with this approach:

* We already set `aria-label` on the `Icon` here. Why is this not enough?
* Adding announcer to just the `MessageStatusIndicator` is likely to lead to end-user confusion: Isn't it the case that this will simply announces "sending message" without saying which message? This could be particularly problematic for "message seen" because an older message might be seen and then the user won't know which message was seen.

The alternative would be to add `area-describedby` to the `MessageStatusIndicator` and use the message status into account when announcing the message (and announce the message again when status changes). So the announcement would be something like:

"Message "Hello world" sent by you at "xxx" was sent"
"Message "Hello world" sent by you at "xxx" was seen by "YY"
etc.

Not sure that's great UX (The "Hello world" part could be long and would become repetitive). My questions are:

* What is the UX we're aiming for here. Are we sure we want the UX we'll get by the current PR?
* Even for this UX, I don't understand why we need `LiveMessage` when we aleady have `aria-label` defined for the `Icon`.

Reply:
Please add a comment in the code with this explanation, so that we know in the future that we needed this for mobile.